.rubocop.yml

```diff
@@ -2,3 +2,10 @@ inherit_gem:
   alpinelab-codestyle:
     - config/default.yml
     - config/rails.yml
+
+AllCops:
+  inherit_mode:
+    merge:
+      - Exclude
+  Exclude:
+    - node_modules/**/*
```

Comment:
All this change can be removed if we bump to a version of the `alpinelab-codestyle` gem that embeds https://github.com/alpinelab/codestyle/pull/46